### PR TITLE
Feature/redo build

### PR DESCRIPTION
**Pull request by toloudis**
Time to review:  It's a ton of changes.

This is a complete redo of this plugin's code structure by starting from scratch with the jupyter lab extension tutorial.

You might take my word for it that most of the files are total boilerplate except for `src/` and `nbvv/` which contain the core implementation.

The main deviation from boilerplate tutorial files is the addition of `less` and `postcss` to adapt antd3's styling.

The current slate of github workflows is very strict but automates a whole lot of things.  

Doing a full release to pypi is of course untested.  Probably should wait until the volume viewer gets a few more key bugfixes.  Also, we need to refactor the "raw data array" codepath (which is how this plugin passes data to the viewer) so that it uses the IVolumeLoader interface.

### CODE REVIEW

File: examples/demo.ipynb [hunk 1]
```diff
@@ -5,6 +5,15 @@
    "execution_count": 1,
    "metadata": {},
    "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Attempted file (/Users/danielt/Downloads/AICS-12_821.ome.tif) load with reader: aicsimageio.readers.bfio_reader.OmeTiledTiffReader failed with error: No module named 'bfio'\n",
```
Comment (blairlyons) [inline examples/demo.ipynb:12]: is this error a problem?

File: nbvv/img_prep.py [hunk 2]
```diff
@@ -33,6 +36,8 @@ def resize(image, output_shape):
 def atlas_dimensions(
     aics_image, max_edge=2048, channel_names=None, physical_pixel_size=(1.0, 1.0, 1.0)
 ):
+    # physicalPixelSize is in xyz order?
```
Comment (blairlyons) [inline nbvv/img_prep.py:39]: is there a check needed for this?

**Review by blairlyons — APPROVED** [review]

File: examples/demo.ipynb (hunk 1 above)
Comment (toloudis) [reply examples/demo.ipynb:12]: No - this is more like a warning related to aicsimageio and really has no bearing on this PR.  I can probably reset the notebook to clear that out.  In the example, that message gets written out but the data still loads.

**Comment by interim17** [comment]
Followed the installation and build instructions in the README and I'm getting some errors, could be user error? One looks like this when i run `jlpm build`:
````
An error occurred.
ValueError: Extensions require a devDependency on @jupyterlab/builder@^3.6.3, you have a dependency on 4.1.6

**Review by interim17 — COMMENTED** [review]
So I'm clear, re: approval criteria this branch should be building and running a working extension.

I'm not super familiar with this repo but I'm seeing build errors and errors in the notebook when I run jupyter lab.

**Review by interim17 — APPROVED** [review]
Thanks for walking me through this!

**Pull request merged**